packages/web/src/scss/components/Modal/_ModalDialog.scss

```diff
@@ -17,6 +17,7 @@
     left: 50%;
     display: flex;
     flex-direction: column;
+    box-sizing: border-box;
```

Comment:
Would you drop a comment to explain why it's necessary? I think it will be helpful, especially when `all: unset` is in another file.